auth/src/config/index.ts

```diff
@@ -2,8 +2,14 @@ import fs from 'fs';
 
 export namespace Config {
     export const DATABASE_PATH = 'database/';
-    export const DATASTORE_READER = `${process.env.INSTANCE_DOMAIN || `http://reader`}:9010`;
-    export const DATASTORE_WRITER = `${process.env.INSTANCE_DOMAIN || 'http://writer'}:9011`;
+    export const DATASTORE_READER = `http://${process.env.DATASTORE_READER_HOST || 'reader'}:${parseInt(
+        process.env.DATASTORE_READER_PORT || '',
```

Comment:
If you haven't concrete defaults, throw an error and termintate the server with a non-zero exit code.